chain/cosmos/tokenfactory.go

```diff
@@ -0,0 +1,87 @@
+package cosmos
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	"github.com/strangelove-ventures/interchaintest/v8/ibc"
+)
+
```

Comment:
Godoc comments on all these functions could be useful. As someone not too familiar with the `tokenfactory` module it's not immediately obvious what the behavior of a few of these would be.

Reply:
Resolved in 27e80ece036f73a7fbb71659634274ae76536020